### DISCUSSION TITLE
Nightly 1.2.0 - CAPS LOCK don't work and some weird problems

### DISCUSSION THREAD

**Discussion by emporiobviver** [rustdesk/rustdesk#2210] [Q&A] [answered]
Hi guys.

My primary language is Brazilian Portuguese, we use Portuguese ABNT2 keyboard layout and installed 1.2 Nightly due accent problems.

But now, when i connect in any machine, i can't use CAPS LOCK to digit in CAPS, i need to hold Shift but even Shift sometimes don't work.

This problem occurs on 1.1.9 and 1.2 versions, on legacy and map modes.

There's a way to solve this problem?

I self-hosting my server with Ubuntu Server 22.10 LTS.
All machines have Windows 10 Pro.
Keyboards layout are Portuguese ABNT2.
All machines have Rust 1.2.0 version running.

Sorry my bad english.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/issues/2211
